eng/pipelines/official.yml

```diff
@@ -1,17 +1,16 @@
 # Pipeline: https://dev.azure.com/dnceng/internal/_build?definitionId=1298
 
-trigger:
-  batch: true
-  branches:
-    include:
-    - main
-    - release/*
-
+# Note: This pipeline can never run automatically because the sourceBranch parameter is required.
+trigger: none
 pr: none
 
 parameters:
-- name: stabilizePackageVersion
-  displayName: Stabilize package version
+- name: sourceBranch
+  displayName: 🚩 Source Branch 🚩
+  type: string
+
+- name: dividerAzDO
+  displayName: '[ ######## AZURE DEVOPS ######## ]'
```

Comment:
I tried a few variation of "group header" concepts in the Azure Pipelines UI. This one was the least gaudy and easy enough to understand at a glance.